src/lib/shared/components/ui/Jumbotron/Jumbotron.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import { renderWithTheme } from '../../fixtures/renderWithTheme';
+import { Jumbotron, TEST_IDS } from './Jumbotron';
+
+describe('Jumbotron', () => {
+    it('should render', () => {
+        const { getByTestId } = renderWithTheme(<Jumbotron headerText="Test" />);
+        const container = getByTestId(TEST_IDS.CONTAINER);
+        expect(container).toBeVisible();
+    });
+    it('should render header text', () => {
+        const { getByText } = renderWithTheme(<Jumbotron headerText='Test text' />);
+        const header = getByText(/test text/i);
```

Comment:
I support leaving the regex expression here because I don't think we have another example of a regex selector anywhere in the app. However, the shortcoming of this selector is that the test will pass regardless of text casing. It could be a poor selector choice for testing text rendering that should be case-sensitive. 

For example, Material UI's base button converts all button text to uppercase letters by default, but our design system does want that behavior. If I were testing our Therify button, I would prefer to test that the button text renders _as expected_ by querying on the cased string (ex `getByText("Test text")`). 

Nothing to change here, just something to be aware of as you continue writing tests 👍